Formula/amr-client.rb

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I removed this line in an amend commit. This review seems to still trigger off blank lines I believe to be desired.  I tricked it to go away by removing all blank lines but it came back after I reverted those changes.